packages/core/scripts/gas-rebate/index.js

```diff
@@ -213,39 +213,50 @@ async function parseRevealEvents({ committedVotes, revealedVotes, priceData, reb
   };
 }
 
+// The UMA dev account sometimes claims rewards on behalf of other voters, to save them gas
+// speed up the reward retrieval, and ensure that no rewards expire. We want to ignore RewardsRetrieved
+// events that arise from such a batch retrieval.
+// Example of such a transaction: https://etherscan.io/tx/0xed907cc499fb6bdccb6fb350dd8dd9cf90e7b24c855a5e857b24156f18e0e4bb#eventlog
+const UMA_DEV_ACCOUNT = "0x9a8f92a830a5cb89a3816e3d267cb7791c16b04d";
+
 async function parseClaimEvents({ claimedRewards, priceData, rebateOutput }) {
   const rewardedVotersToRebate = {};
 
   const progressBarClaim = multibar.create(claimedRewards.length, 0, { label: "Claim Events" });
 
   for (let i = 0; i < claimedRewards.length; i++) {
     const claim = claimedRewards[i];
-    const voter = claim.returnValues.voter;
-    const roundId = claim.returnValues.roundId;
-    const identifier = web3.utils.hexToUtf8(claim.returnValues.identifier);
-    const requestTime = claim.returnValues.time;
     const [transactionBlock, transactionReceipt] = await Promise.all([
       web3.eth.getBlock(claim.blockNumber),
       web3.eth.getTransactionReceipt(claim.transactionHash)
     ]);
-    const gasUsed = parseInt(transactionReceipt.gasUsed);
-    const txnTimestamp = transactionBlock.timestamp;
+    // Check if claim txn was sent by an UMA dev batch retrieval.
+    if (transactionReceipt.from !== UMA_DEV_ACCOUNT) {
+      const voter = claim.returnValues.voter;
+      const roundId = claim.returnValues.roundId;
+      const identifier = web3.utils.hexToUtf8(claim.returnValues.identifier);
+      const requestTime = claim.returnValues.time;
+
+      const gasUsed = parseInt(transactionReceipt.gasUsed);
+      const txnTimestamp = transactionBlock.timestamp;
+
+      const key = `${voter}-${roundId}-${identifier}-${requestTime}`;
+      const val = {
+        voter,
+        roundId,
+        requestTime,
+        identifier,
+        claim: {
+          transactionBlock: transactionBlock.number,
+          hash: transactionReceipt.transactionHash,
+          gasUsed,
+          txnTimestamp
+        }
+      };
```

Comment:
How hard would it be to print out a log for each transaction that was ignored? Just so the runner can have some assurance that the script is ignoring the right things.

Reply:
I did do this but in a more roundabout way. Expand the "script details" in the Details section of the PR description.  Notice how there were 233 claim events processed, but only 157 got rebated. The diff is due to the ignores.

If its clearer I could also keep a runnning count of each claim txn ignored and print that at the end

Reply:
I see. Yeah, that makes sense. Didn't realize you could see that difference there. Yeah, I think that's enough for a sanity check.